test/videojs-flashls-source-handler.test.js

```diff
@@ -5,6 +5,29 @@ import sinon from 'sinon';
 import videojs from 'video.js';
 
 import handler from '../src/index';
+import { updateAudioTrack, onTrackChanged } from '../src/flashlsAudioTracks.js';
+
+const noop = () => {};
+
+/* eslint-disable camelcase */
+const makeMochTech = (getters, setters) => {
```

Comment:
since this function (and `noop`) are used in both this file and the representations.test.js It would be a good idea to move the two functions to their own file like `utils.js` and then just `import makeMochTech from './utils.js';` both here and in the representations test file